packages/composite-checkout/src/wpcom/wp-checkout-order-review.js

```diff
@@ -11,7 +11,12 @@ import styled from '@emotion/styled';
 import joinClasses from './join-classes';
 import Coupon from './coupon';
 import WPTermsAndConditions from './wp-terms-and-conditions';
-import { useLineItems, renderDisplayValueMarkdown, OrderReviewLineItems, OrderReviewTotal, OrderReviewSection } from '../public-api';
+import { useLineItems, renderDisplayValueMarkdown } from '../public-api';
+import {
+	OrderReviewLineItems,
```

Comment:
I like what you've done here, but can we rename these to start with `WP...`? I can see an argument for these simpler names, but until we move all these files to another place I'd worry about confusion where two components in one package that have the same name but different APIs.

Reply:
Good call. I can change the names.

Reply:
Updated!